README.md

```diff
@@ -188,10 +188,11 @@ Learn more about using
 ## Badge
 
 Show the world you're using _Jest_ →
-[![tested with jest](https://img.shields.io/badge/tested_with-jest-99424f.svg)](https://github.com/facebook/jest)
+[![tested with jest](https://img.shields.io/badge/tested_with-jest-99424f.svg)](https://github.com/facebook/jest) [![jest](https://cdn.rawgit.com/aleen42/badges/master/src/jest_2.svg)](https://github.com/facebook/jest)
```

Comment:
Should the file be hosted on the jest website?

Reply:
@SimenB Sure, where is the repository of the jset website?

Reply:
It's in the website folder in this repo.

Reply:
To be specific: https://github.com/facebook/jest/tree/master/website/static/img